model/project_event.go

```diff
@@ -128,43 +191,23 @@ func (p *ProjectChangeEvents) RedactGitHubPrivateKey() {
 	}
 }
 
-// RedactPrivateVars redacts private variables from the project modification event.
-func (p *ProjectChangeEvents) RedactPrivateVars() {
+// RedactPrivateVars redacts project variables changes from the project
+// modification event.
+// TODO (DEVPROD-9384): this can be removed entirely once project event logs are
+// migrated to not store any project var values. Project change events should
+// already redact all variable values when the log is inserted into the DB (see
+// redactVars).
```

Comment:
nit: capitalize + can we specify we're referring to `ProjectChangeEvent.RedactVars`